client/src/components/form/index.js

```diff
@@ -72,6 +82,10 @@ export const Form = ({ initialValues, isDisabled }) => {
               <Fields isDisabled={isDisabled} />
             </Box>
 
+            {Object.keys(errors).length !== 0 && <Typography className={classes.title} noWrap>
+              Your form contains errors.
```

Comment:
We should get some wording for this from Rachit - I feel like he'll want "Please review" or something included as well

Reply:
@fresh-rachit thoughts?